test/testdata/packager/export_for_test/foo/foo.rb

```diff
@@ -27,9 +27,6 @@ def self.stub_stuff!; end
 
   Opus::Util::Nesting::Public.public_method
 
-  # util/__package.rb exposed via export_for_test, cannot access from here:
-  Opus::Util::Nesting.nesting_method
```

Comment:
Rather than delete this, what do you think about changing the test to assert the new expected behavior?

Reply:
Fixed